scripts/aws.sh

```diff
@@ -0,0 +1,488 @@
+#!/bin/bash
+
+set -uo pipefail
+
+export AWS_PAGER=""
```

Comment:
Please, add a comment to explain why we need this.

Reply:
I removed it in the end.